pkg/kando/location_pull.go

```diff
@@ -91,28 +62,6 @@ func locationPull(ctx context.Context, p *param.Profile, path string, target io.
 	return location.Read(ctx, target, *p, path)
```

Comment:
Can we remove locationPull and locationPush helper methods in kando packages? Do we need them anymore here?

Reply:
right now it's being used in `location_test.go` file. We will have to refactor the test file and add new tests. Will raise a new PR for that